chart/templates/NOTES.txt

```diff
@@ -187,3 +187,9 @@ Information on how to set a static webserver secret key can be found here:
 https://airflow.apache.org/docs/helm-chart/stable/production-guide.html#webserver-secret-key
 
 {{- end }}
+
+{{- if or .Values.postgresql.postgresqlUsername .Values.postgresql.postgresqlPassword }}
+
+   {{ fail "postgresql.postgresqlUsername and postgresql.postgresqlPassword have been renamed to postgresql.auth.username and postgres.auth.password }}
```

Comment:
```suggestion
   {{ fail "postgresql.postgresqlUsername and postgresql.postgresqlPassword have been renamed to postgresql.auth.username and postgres.auth.password" }}
```

Looks like this is causing the unit test failures. Can you test this part out manually as well? Thanks.